src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -219,11 +219,14 @@ namespace HighTempRadiantSystem {
 
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		static bool GetInputFlag( true ); // First time, input is "gotten"
```

Comment:
@RKStrand Now that there's a unit test for high temp radiant, this needs to move up to anonymous namespace and there needs to be a HighTempRadiant::clear_state function.  Let me know if you need an example to follow.